tests/ext/bottle/views/index.tpl

```diff
@@ -0,0 +1,7 @@
+%if name == 'World':
+    <h1>Hello {{name}}!</h1>
+    <p>This is a test.</p>
+%else:
+    <h1>Hello {{name.title()}}!</h1>
+    <p>How are you?</p>
+%end
```

Comment:
Can we format this?

Reply:
sorry, do you mean a new line at the end?

Reply:
right.

Reply:
sure